app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable, which means it is shared across all instances of `Animal`. This can lead to unintended behavior, especially in a multi-threaded environment. Consider using an instance variable or another approach to track alive animals.